app/js/tabmanager.js

```diff
@@ -237,16 +237,16 @@ const TabManager = {
       return;
     } else if (typeof tabOrTabId === "number") {
       tabId = tabOrTabId;
+      TabManager.tabTimes[String(tabId)] = new Date().getTime();
     } else {
       tabId = tabOrTabId.id;
+      TabManager.tabTimes[String(tabId)] = tabOrTabId?.lastAccessed ?? new Date().getTime();
     }
 
     if (typeof tabId !== "number") {
       console.log("Error: `tabId` is not an number", tabId);
       return;
     }
-
-    TabManager.tabTimes[String(tabId)] = new Date().getTime();
```

Comment:
Moving this above the check on 244 changes the logic of this function. If `tabID` is not a number, like can be the case with tabs that have been replaced (I believe..), then this function should not do anything.

Reply:
.

Reply:
Sorry for the delay, was very busy recently. I'll make adjustments shortly.

Reply:
Hi, this seems like it'd be a valuable addition, do you have time to update the code?